tensorflow/lite/delegates/xnnpack/binary_elementwise_tester.cc

```diff
@@ -246,7 +270,9 @@ std::vector<char> BinaryElementwiseTester::CreateTfLiteModel(
     }
   }
 
-  float input2_scale = 0;
+  std::vector<float> input2_scales;
+  std::vector<int64_t> input2_zero_points;
+  int32_t input2_quantized_dimension;
```

Comment:
Initialize to 0, as this is the default value in `CreateQuantizationParameters`

Reply:
Fixed